x/lockup/keeper/lock.go

```diff
@@ -135,6 +135,30 @@ func (k Keeper) CreateLock(ctx sdk.Context, owner sdk.AccAddress, coins sdk.Coin
 	return lock, nil
 }
 
+// CreateLockNoSend behaves the same as CreateLock, but does not send the coins to the lockup module account.
+func (k Keeper) CreateLockNoSend(ctx sdk.Context, owner sdk.AccAddress, coins sdk.Coins, duration time.Duration) (types.PeriodLock, error) {
```

Comment:
Would it be possible to move bank send to `CreateLock`, make everything no send by default and from `CreateLock` call `CreateLockNoSend` and then perform the bank send?

Reply:
This is a great suggestion, implementing now

Reply:
lockNoSend (with a warning about expectation from caller) done here https://github.com/osmosis-labs/osmosis/pull/5162/commits/d783cd5bcb795694c0dbbdf339b08604af12a8fb

Note: I think its important to use the lockNoSend nomenclature in place of lock since the expectation of the previous lock function involved sending the coins.